### PR TITLE
First Microverse Project

### PR DESCRIPTION
**Pull request by Willian-Berrocal**
> Created a branch for working on it as the GitHub flow states.
> Added the linter's configuration and assured that it works with GitHub actions.
> .gitignore file added and used to ignore non-important files in the repo.
> HTML and CSS files added, as well as the test.md (file ignored by git).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
```
Comment (franclobo) [inline README.md]: - [x] Kindly replace the information on the README template with the information on your project. 👇🏽 

> Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- Hello Microverse could be a good title.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```
Comment (franclobo) [inline README.md]: - [x] Kindly replace the information on the README template with the information on your project. 👇🏽 

> Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- HTML & CSS could be an option.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
```
Comment (franclobo) [inline README.md]: - [x] Please 🙏🏽 remove those sections that you are not using yet for a professional README file.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```
Comment (franclobo) [inline README.md]: - [x] Kindly replace the information on the README template with the information on your project. 👇🏽 

> Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- Clone this repository and open it in your browser could be a good instruction.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
```
Comment (franclobo) [inline README.md]: - [x] Please 🙏🏽 remove those sections that you are not using yet for a professional README file.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (franclobo) [inline README.md]: - [x] Kindly replace the information on the README template with the information on your project. 👇🏽 

> Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- We could use our name in the Authors section.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (franclobo) [inline README.md]: - [x] Kindly replace the information on the README template with the information on your project. 👇🏽 

> Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- Microverse is a good option.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (franclobo) [inline README.md:47]: - [x] Please 🙏🏽 add a LICENSE file to your repository and make sure that the link works as expected.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Willian-Berrocal, 

# STATUS: Required Changes ♻️
Good job so far!
It is good at first 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Correct GitHub flow ✔️ 
- Descriptive commits ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please 🙏🏽 add a title and a description of the activities in your project in your pull request.

> Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is my first project in Microverse
```
Comment (mgmediaweb) [inline README.md]: Kindly, I suggest you write a summary of your project. stating that it is your first project is not as relevant to a reviewer in the future

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Willian-Berrocal , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost4

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️

*** Kindly, I suggest you indicate in your Pull Request a summary of what you are presenting. putting that it is your first project, is not so relevant for a reviewer

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Willian-Berrocal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**